sbws/config.default.ini

```diff
@@ -87,6 +87,8 @@ min_relays = 50
 to_file = yes
 # Whether or not to log to stdout
 to_stdout = no
+# Whether or not to log to syslog
+to_syslog = no
```

Comment:
Please add to_syslog_level defaulting to `${level}`.

Please add to_syslog_format (feel free to have it default to the string you like instead of `${format}`

Reply:
i added then removed before opening this PR cause thought those settings doesn't probably need to be modified by users, however i'm not sure and it's fine for me to add them